tests/beaker_tests/cisco_bgp_af/test_bgpaf.rb

```diff
@@ -208,7 +208,7 @@
   },
 }
 
-redistribute = Array[%w(static RouteMap), %w(lisp RouteMap)]
+redistribute = Array[%w(lisp RouteMap), %w(static RouteMap)]
```

Comment:
The redist getter return values were different on n6k (lex.sort) vs n9k (unsorted) so I added a sort to the NU getter to normalize the data.
The array above is used by puppet resource so it has to be in sorted order to match the getter output.